.release/ci.hcl

```diff
@@ -13,10 +13,7 @@ project "nomad" {
 
     release_branches = [
       "main",
-      "release/1.0.x",
-      "release/1.1.x",
-      "release/1.2.x",
-      "release/1.3.x",
+      "release/**",
```

Comment:
Won't this trigger release workflows on dead branches? Ex. https://github.com/hashicorp/nomad/branches/all?query=release%2F

Reply:
The pipeline won't be exercised until a new commit is made to the branch. When I say pipeline, I mean all of the common workflows that run after a commit is made on a release branch (not including the promotions to staging/production, which still require a manual trigger using `bob`).